+Prefs/Numeric.m

```diff
@@ -0,0 +1,17 @@
+classdef Numeric < Base.pref
```

Comment:
what is this used for?

Reply:
Basically, to tell other things that subclasses of this are 'scannable', i.e. I can put in numeric values, or poll min/max without things breaking.